docs/patterns.json

```diff
@@ -85,17 +85,20 @@
     {
       "patternId": "psavoidusingusernameandpasswordparams",
       "level": "Error",
-      "category": "ErrorProne"
+      "category": "Security",
+      "subcategory": "Cryptography"
```

Comment:
Given this is related to username and password, maybe Auth would be more appropriate. What do you think?

Reply:
You're right 👍 changed it